### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by aniljadhavmca**
Added unique script handle
Changed slug
Improved script dependency 
Escape the variables
Removed commented code

### CODE REVIEW

File: includes/scripts.php [hunk 1]
```diff
@@ -5,62 +5,58 @@
...
 		} else {
-			$publishable = $stripe_options['live_publishable_key'];
+			$publishable = $ssp_options['live_publishable_key'];
 		}
 
-		wp_enqueue_style('style', STRIPE_BASE_URL . 'assets/css/style.css');
-		wp_enqueue_script('jquery');
-		wp_enqueue_script('stripe', 'https://checkout.stripe.com/checkout.js');
-		wp_register_script( 'stripe_processing', STRIPE_BASE_URL . 'includes/js/stripe-processing.js');
+		wp_enqueue_style('style', SSP_BASE_URL . 'assets/css/style.css');
```
Comment (Nikschavan) [inline includes/scripts.php]: change the handle of the css file to be more unique

File: includes/scripts.php [hunk 2]
```diff
@@ -5,62 +5,58 @@
...
+		wp_register_script( 'ssp_processing', SSP_BASE_URL . 'includes/js/stripe-processing.js', array('jquery'));
```
Comment (Nikschavan) [inline includes/scripts.php]: use `-` instead of `_` for script handle

File: includes/scripts.php [hunk 3]
```diff
@@ -5,62 +5,58 @@
...
-				background:<?php echo $color ?>;
-				color:<?php echo $title_color ?>;;
+				background:<?php _e( $color ); ?>;
+				color:<?php _e( $title_color ); ?>;
 			}
 			#bsfStripeButton:hover {
-				background:<?php echo $hovercolor ?>;
-				color:<?php echo $title_hovercolor ?>;;
+				background:<?php _e( $hovercolor ); ?>;
+				color:<?php _e( $title_hovercolor ); ?>;;
```
Comment (Nikschavan) [inline includes/scripts.php]: These variables should be passed through `esc_attr()`

File: simple-stripe-payments.php [hunk 4]
```diff
@@ -41,80 +41,80 @@
...
-function add_ajax_actions() {
-    add_action( 'wp_ajax_bsf_create_payment', 'bsf_create_payment_callback' ); 
-	add_action( 'wp_ajax_nopriv_bsf_create_payment', 'bsf_create_payment_callback' );
+function ssp_add_ajax_actions() {
+    add_action( 'wp_ajax_ssp_create_payment', 'ssp_create_payment_callback' ); 
+	add_action( 'wp_ajax_nopriv_ssp_create_payment', 'ssp_create_payment_callback' );
 }
 
-add_action( 'init', 'add_ajax_actions' );
+add_action( 'init', 'ssp_add_ajax_actions' );
```
Comment (Nikschavan) [inline simple-stripe-payments.php]: No need for a separate function `ssp_add_ajax_actions()` to register ajax actions

File: includes/backend-scripts.php [hunk 5]
```diff
@@ -5,10 +5,10 @@
...
 }
 
-add_action( 'admin_enqueue_scripts', 'color_picker_assets' );
+add_action( 'admin_enqueue_scripts', 'ssp_color_picker_assets' );
 
-function color_picker_assets($hook_suffix) {
+function ssp_color_picker_assets() {
 	    wp_enqueue_style( 'wp-color-picker' );
-	    wp_enqueue_script( 'my-script-handle', plugins_url('../assets/js/custom.js', __FILE__  ), array( 'wp-color-picker' ), false, true );
 	    wp_enqueue_style( 'custom-css' , plugins_url('../assets/css/backend-style.css', __FILE__  ) );
```
Comment (Nikschavan) [inline includes/backend-scripts.php]: change the handel

File: includes/scripts.php (hunk 1 above)
Comment (aniljadhavmca) [reply includes/scripts.php]: Changed CSS Handle - https://goo.gl/355Hrn

File: includes/scripts.php (hunk 2 above)
Comment (aniljadhavmca) [reply includes/scripts.php]: Replaced with `-`

File: includes/backend-scripts.php (hunk 5 above)
Comment (aniljadhavmca) [reply includes/backend-scripts.php]: Changed CSS Handel - https://goo.gl/kRmuUu

File: includes/scripts.php (hunk 3 above)
Comment (aniljadhavmca) [reply includes/scripts.php]: Variables passed through esc_attr() - https://goo.gl/m2UnHQ

File: simple-stripe-payments.php (hunk 4 above)
Comment (aniljadhavmca) [reply simple-stripe-payments.php]: Removed separate function for registering action - https://goo.gl/hN3hVH

**Review by Nikschavan — APPROVED** [review]

**Pull request merged**